packages/medusa/src/loaders/medusa-app.ts

```diff
@@ -141,15 +142,10 @@ export async function revertMedusaApp({
   })
 }
 
-export const loadMedusaApp = async (
+export const loadModules = async (
```

Comment:
From what I saw was this method was just loading the modules and rest of the stuff like loading routes, workflows are happened somewhere else.

Maybe I am wrong, but I thought `loadingApp` means, loading everything and not just modules :)

Reply:
MedusaApp is a generic concept that can be run outside of the core as well, like a serverless func. I know the naming is confusing because the core can be considered an app as well, but this is a different concept. No strong feelings about the naming of this, just thought I'll put it out there. :)

Reply:
Yes indeed, it is possible to spinup the modules without going through the medusa package, which in fact is just holding the http layer and loaders. But i am fine with that for now

Reply:
I have no strong feelings about the name either :). But, a bit confused with naming and what exactly this method is supposed to do 🤔

Reply:
To help you a bit with the naming: `MedusaApp` is the real "core" loading the modules (internal and soon external), setting up the links between them, and exposing a Remote Query and Remote Link instances.
Medusa (@medusajs/medusa) is the ecommerce application using all the modules, using all the workflows from `@medusajs/core-flows`, exposing a REST Api and loading custom things.

Reply:
Okay. That makes sense.